src/components/App/App.jsx

```diff
@@ -1,5 +1,29 @@
+import logoIconik from "../../assets/images/logo.png";
+import './app.css';
+
 const App = () => (
-  <h1>El salon se llama iconik</h1>
+  <>
+    <section className="menu">
+      <img className='menu__logoIconik' src={logoIconik} alt="Iconik" />
```

Comment:
the className `menu__logoIconk` can't be camelcase, so please update it to `menu__logo-iconik`